jaxrs-clients/src/main/java/com/palantir/remoting2/jaxrs/OkHttpClients.java

```diff
@@ -0,0 +1,85 @@
+/*
+ * Copyright 2016 Palantir Technologies, Inc. All rights reserved.
+ */
+
+package com.palantir.remoting2.jaxrs;
+
+import com.google.common.collect.ImmutableList;
+import com.google.common.net.HttpHeaders;
+import com.palantir.remoting2.clients.CipherSuites;
+import com.palantir.remoting2.clients.ClientConfig;
+import com.palantir.remoting2.config.service.BasicCredentials;
+import com.palantir.remoting2.config.service.ProxyConfiguration;
+import com.palantir.remoting2.config.service.ServiceConfiguration;
+import com.palantir.remoting2.config.ssl.TrustContext;
+import com.palantir.remoting2.tracing.okhttp3.OkhttpTraceInterceptor;
+import java.util.concurrent.TimeUnit;
+import okhttp3.ConnectionPool;
+import okhttp3.ConnectionSpec;
+import okhttp3.Credentials;
+import okhttp3.OkHttpClient;
+import okhttp3.TlsVersion;
+
+public class OkHttpClients {
+    private OkHttpClients() {}
+
+    public static OkHttpClient createClient(ServiceConfiguration serviceConfig) {
+        ClientConfig config = ClientConfig.fromServiceConfig(serviceConfig);
+        return createClient(config);
+    }
+
+    public static OkHttpClient createClient(ClientConfig config) {
```

Comment:
thoughts on moving this to a separate project (okhttp-clients) and making jaxrsclients and retrofitclients depend on it, so that we can remove duplication from Retrofit2ClientBuilder and FeignJaxRsClientBuilder?